packages/chat-client/src/controllers/engine.ts

```diff
@@ -187,12 +190,9 @@ export class ChatEngine extends IChatEngine {
     ZAccount.parse(account);
 
     const identityKey = await this.registerIdentity(account, onSign);
-    await this.registerInvite(account, false);
 
     this.currentAccount = account;
 
-    await this.subscribeToSelfInviteTopic();
-
     if (this.client.syncClient) {
       if (this.client.syncClient.signatures.keys.includes(account)) {
         const { signature } = this.client.syncClient.signatures.get(account);
```

Comment:
Does this means that we are storing signatures ?

Reply:
Yes, to re-init the stores without the need for another `register` call in the sync client

Reply:
yes, but even if you do not store the `signature`, you could sync to their stores via accessing the `storeMap` which has topics